### ISSUE TITLE
Feature request: isotopic multiplexing

### ISSUE THREAD

**Issue by cutleraging** [MannLabs/directlfq#55]
Hello,

I am using DIA-NN to process multiplexed samples using dimethyl labeling. I would like to request a feature where your program can be used to derive the protein intensities for each of the channels within a sample. Please let me know if you need an example of this.

Best,
Ronnie

**Comment by ammarcsj** [comment]
Hi Ronnie,

thanks for your comment. Please feel free to check out our RefQuant package: https://github.com/MannLabs/refquant

It is designed to do exactly what you describe. There is no GUI for it, but there are example notebooks that you can follow.

Best
Constantin

**Comment by cutleraging** [comment]
Right, I was aware of that. But does it produce directLFQ for quantification?

**Comment by ammarcsj** [comment]
Yes, the .protein_intensities.tsv table is directLFQ quantified

**Comment by cutleraging** [comment]
This is great! Thanks!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
